src/routing_header.ts

```diff
@@ -49,7 +49,7 @@
  * @param {Object} params - the request header parameters.
  * @return {string} the routing header value.
  */
-export function fromParams(params) {
+export function fromParams(params: {}) {
```

Comment:
Can you annotate the return type here and other exported functions?

Reply:
I got as many as I could.  I'll be making more of these PRs to tighten things up.